### ISSUE TITLE
why?

### ISSUE THREAD

**Issue by RaganrokV** [locuslab/TCN#74]
I change the output "return torch.mean(self.network(x),dim=2)" for multi-feature time series and the training time reduces significantly..so why--lol
